build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -0,0 +1,494 @@
+// <copyright file="GenerateIntegrationDefinitions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.Loader;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PrepareRelease
+{
+    public static class GenerateIntegrationDefinitions
+    {
+        const string InstrumentMethodAttributeName = "Datadog.Trace.ClrProfiler.InstrumentMethodAttribute";
+        const string InterceptMethodAttributeName = "Datadog.Trace.ClrProfiler.InterceptMethodAttribute";
+        public static void Run(ICollection<string> assemblyPaths, params string[] outputDirectories)
```

Comment:
whitespace nitpicking 
```suggestion
        const string InterceptMethodAttributeName = "Datadog.Trace.ClrProfiler.InterceptMethodAttribute";

        public static void Run(ICollection<string> assemblyPaths, params string[] outputDirectories)
```